estudios_socioeconomicos/utils.py

```diff
@@ -6,9 +6,15 @@ def _delete_file(path):
     Parameters:
     -----------
         path : A string that specify the location of the file.
+
+    Returns
+    -----------
+    A boolean value. True if the path was deleted, False otherwise.
     """
     if os.path.isfile(path):
         os.remove(path)
+        return os.path.isfile(path)
```

Comment:
This will return False, if the file is properly deleted. So this function as it is right now, will always return False.